backend/driver.pm

```diff
@@ -54,20 +49,20 @@ sub start {
     my ($self) = @_;
 
     my $p1, my $p2;
-    pipe($p1, $p2) or die "pipe: $!";
+    pipe($p1, $p2) or bmwqemu::logdie("pipe: $!");
     $self->{from_parent} = $p1;
     $self->{to_child}    = $p2;
 
     $p1 = undef;
     $p2 = undef;
-    pipe($p1, $p2) or die "pipe: $!";
+    pipe($p1, $p2) or bmwqemu::logdie("pipe: $!");
     $self->{to_parent}  = $p2;
     $self->{from_child} = $p1;
 
-    printf STDERR "$$: to_child %d, from_child %d\n", fileno($self->{to_child}), fileno($self->{from_child});
+    bmwqemu::fctwarn sprintf "$$: to_child %d, from_child %d\n", fileno($self->{to_child}), fileno($self->{from_child});
 
     my $pid = fork();
-    die "fork failed" unless defined $pid;
+    bmwqemu::logdie "fork failed" unless defined $pid;
```

Comment:
IMHO you are now loosing the different logging contexts we should preserve. If this dies here it is not something that the test reviewer is interested in but a backend developer